index.html

```diff
@@ -4,10 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Personal Portfolio</title>
+    <title>Document</title>
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    
+    <section id="Home">
+        <nav>
+            <div class="logo">
+                <img src="" alt="">
+                <h2>My logo</h2>
+            </div>
+            <div class="navbar_links">
+                <a href="#portfolio">Portfolio</a>
+                <a href="#about">About</a>
+                <a href="#contact">Contact</a>
+                <a href="#"><img src="assets/Disabled.png" alt=""></a>
+            </div>
+            <div class="phone_navbar">
+                <img src="assets/Icon - Menu.png" alt="">
+            </div>
+        </nav>
+
+        <div class="headline">
+            <div class="social_links">
+                <img src="assets/github.svg" alt="">
+                <img src="assets/Linkedin icon.svg" alt="">
+                <img src="assets/angel list.svg" alt="">
+                <img src="assets/twitter.svg" alt="">
+                <img src="assets/medium.svg" alt="">
+            </div>
+            <div class="description">
+                <h1>Hey There <br> I'm Alex</h1>
+                <h2>I'm a Software Developer</h2>
+                <p>I can help you build a product , feature or website.
+                     Look through some of my work and experience! 
+                     If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</p>
+            </div>
+            <!-- <div class="bg_image">
```

Comment:
- [ ] Kindly, add the social media link as it has it is prescribed in the design to have a pixel-perfect design.

<img width="230" alt="right" src="https://user-images.githubusercontent.com/41163571/227023691-54c85470-7663-4398-9b08-5e0d174fae08.png">